tools/codegen/base/syft.json

```diff
@@ -0,0 +1,20 @@
+{
+  "repository": "https://github.com/anchore/syft",
+  "tag_prefix": "v",
+  "bin": "${package}${exe}",
+  "version_range": ">= 0.83.0",
+  "platform": {
+    "x86_64_linux_gnu": {
+      "asset_name": "${package}_${version}_linux_amd64.tar.gz"
+    },
+    "x86_64_macos": {
+      "asset_name": "${package}_${version}_darwin_amd64.tar.gz"
+    },
+    "x86_64_windows": {
+      "asset_name": "${package}_${version}_windows_amd64.zip"
+    },
+    "aarch64_linux_musl": {
+      "asset_name": "${package}_${version}_linux_arm64.tar.gz"
+    }
```

Comment:
- Why is x86_64_linux gnu but aarch64_linux is musl?
  (I guess both binaries would work with musl since syft is written in go.)
- soft seems to provide aarch64_macos binaries. (${package}_${version}_darwin_arm64.tar.gz)

Reply:
> Why is x86_64_linux gnu but aarch64_linux is musl?

They use `goreleaser` to build their artifacts.

Here is the configuration for linux

https://github.com/anchore/syft/blob/2c5d64ac9ed386c387237e71139f95336639da4a/.goreleaser.yaml#L11-L30

They provide the flag `-extldflags '-static'` meaning to statically link the lib used.

So I think we would provide both `gnu` & `musl` ?

> soft seems to provide aarch64_macos binaries.

I would add it tomorrow

Reply:
IIRC, on Linux, go usually calls syscall directly without going through libc, so it should work on both gnu and musl (especially if statically linked).